core/config/general_config.go

```diff
@@ -1318,41 +1316,6 @@ func (*generalConfig) GlobalMinimumContractPayment() (*assets.Link, bool) {
 	}
 	return val.(*assets.Link), ok
 }
-func (*generalConfig) GlobalOCRContractTransmitterTransmitTimeout() (time.Duration, bool) {
```

Comment:
These were actually duplicated with the ones in ocr_config after a develop merge